### PR TITLE
Fix/0041

### PR DESCRIPTION
**Pull request by avniculae**

### CODE REVIEW

File: src/components/containers/ConnectedBorrowForm/ConnectedBorrowForm.tsx [hunk 1]
```diff
@@ -150,8 +150,8 @@ const ConnectedBorrowForm: React.FunctionComponent<ConnectedBorrowFormProps> = (
       />
       <SwapInfo
         balance={form.selectedFixedDebt}
+        currentPositionMarginRequirement={form.margin}
```
Comment (ssbarbee) [inline src/components/containers/ConnectedBorrowForm/ConnectedBorrowForm.tsx:153]: prop positionMargin just below has same value... why not use that prop?

File: src/contexts/MintBurnFormContext/MintBurnFormContext.tsx [hunk 2]
```diff
@@ -490,6 +494,27 @@ export const MintBurnFormProvider: React.FunctionComponent<MintBurnFormProviderP
       }
     }
 
+    // Check that the input margin is >= minimum required margin if removing margin
+    if (
```
Comment (ssbarbee) [inline src/contexts/MintBurnFormContext/MintBurnFormContext.tsx:498]: whats up with the nesting of ifs... you can merge 2 of the ifs into 1. Also declare a const variable telling us what exactly this logic does...

If you name it well you can remove the comments

File: src/contexts/SwapFormContext/SwapFormContext.tsx [hunk 3]
```diff
@@ -764,7 +764,7 @@ export const SwapFormProvider: React.FunctionComponent<SwapFormProviderProps> =
     // Check that the input margin is >= minimum required margin if removing margin
     if (
```
Comment (ssbarbee) [inline src/contexts/SwapFormContext/SwapFormContext.tsx:765]: this is duplicate... also found in the MintForm. Create a utility and share it across both components.

**Review by ssbarbee — CHANGES_REQUESTED** [review]

**Comment by ssbarbee** [comment]
General comment: branch name doesn't reflect anything that you are working on. There is no PR description to give the reviewer indication on what to look at. It is good practice to provide some context for the reviewer.

File: src/components/containers/ConnectedBorrowForm/ConnectedBorrowForm.tsx (hunk 1 above)
Comment (avniculae) [reply src/components/containers/ConnectedBorrowForm/ConnectedBorrowForm.tsx:153]: actually that's not even used for the borrow product; will delete

**Comment by avniculae** [comment]
Branch name refers to the bug number (from Discord) that it's addressing

**Comment by ssbarbee** [comment]
> Branch name refers to the bug number (from Discord) that it's addressing

fair enough link the discord channel to the description of ticket next time around

**Review by ssbarbee — APPROVED** [review]

**Pull request merged**